packages/web/modals/validator-squad.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import classNames from "classnames";
+import { observer } from "mobx-react-lite";
+import { FunctionComponent } from "react";
+import { useMemo } from "react";
+
+import { Icon } from "~/components/assets";
+import { CloseButton } from "~/components/buttons";
+import { SearchBox } from "~/components/input";
+import { IS_FRONTIER } from "~/config/index";
+import { ModalBaseProps } from "~/modals/base";
+
+export const ValidatorSquadModal: FunctionComponent<ModalBaseProps> = observer(
+  (props) => <ValidatorSquadContent {...props} />
+);
+
+const data = [
+  ["Cosmostation", "0.00", "8.44%", "5%"],
+  ["Figment", "0.00", "8.44%", "5%"],
+  ["Stargaze", "0.00", "8.44%", "5%"],
+  ["Frens", "0.00", "8.44%", "5%"],
+  ["Figment", "0.00", "8.44%", "5%"],
+  ["interchain.fm", "0.00", "8.44%", "5%"],
+  ["imperator.co", "0.00", "8.44%", "5%"],
+  ["Chorus One", "0.00", "8.44%", "5%"],
+  ["Electric", "0.00", "8.44%", "5%"],
+  ["wosmongton", "0.00", "8.44%", "5%"],
+];
+
+type Validator = {
+  validator: string;
+  myStake: number;
+  votingPower: number;
+  comissions: number;
+};
+
+interface ValidatorSquadContentProps {
+  onRequestClose: () => void;
+  isOpen: boolean;
+}
+
+const ValidatorSquadContent = observer(
```

Comment:
If you look elsewhere in the app. The component signature is typically:

`const Component: FunctionComponent<{ prop1: string }> = observer(({ prop1 }) => <div/>);`